VideoLocker/res/layout/fragment_user_profile.xml

```diff
@@ -0,0 +1,181 @@
+        android:orientation="vertical">
+
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:background="@color/edx_brand_primary_base"
+            android:gravity="center"
+            android:orientation="vertical"
+            android:padding="@dimen/edx_margin">
+
+            <de.hdodenhof.circleimageview.CircleImageView
+                android:id="@+id/profile_image"
+                style="@style/profile_image"
+                android:layout_width="166dp"
+                android:layout_height="166dp"
+                app:civ_border_width="3dp"
+                tools:src="@drawable/app_icon" />
+
+            <TextView
+                android:id="@+id/username_text"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginBottom="@dimen/widget_margin"
+                android:layout_marginTop="@dimen/widget_margin"
+                android:textColor="@color/white"
+                android:textSize="@dimen/edx_xx_large"
+                tools:text="username" />
+
+            <LinearLayout
+                android:id="@+id/profile_header_content"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:gravity="center"
+                android:orientation="vertical">
+
+                <LinearLayout
+                    android:id="@+id/language_container"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:gravity="center"
+                    android:orientation="horizontal">
+
+                    <org.edx.mobile.third_party.iconify.IconView
+                        android:layout_width="@dimen/edx_x_small"
+                        android:layout_height="@dimen/edx_x_small"
+                        android:layout_marginEnd="@dimen/discussion_posts_icon_margin"
+                        android:layout_marginRight="@dimen/discussion_posts_icon_margin"
+                        app:iconColor="@color/edx_brand_primary_x_light"
+                        app:iconName="fa_comment" />
+
+                    <TextView
+                        android:id="@+id/language_text"
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:textColor="@color/edx_brand_primary_x_light"
+                        android:textSize="@dimen/edx_x_small"
+                        tools:text="English" />
+                </LinearLayout>
+
+
+                <LinearLayout
+                    android:id="@+id/location_container"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="4dp"
```

Comment:
If we really need a 4 DIP margin instead of 5 here, then maybe this should be externalized to a standard dimen resource as well.